test/fs.test.js

```diff
@@ -50,6 +50,15 @@ describe('FileSystem based storage provider', function() {
       });
     });
 
+    it('should return an empty list of containers - promise', function(done) {
+      client.getContainers()
+        .then(function(containers) {
+          assert.equal(0, containers.length);
```

Comment:
We should switch `0` and `containers.length` since the first argument is supposed to be the actual value and the 2nd expected (see https://nodejs.org/api/assert.html#assert_assert_equal_actual_expected_message). I think there's a few more places where this is applicable.

Reply:
Good catch +1 ^